README.md

```diff
@@ -52,8 +52,10 @@ https://iree.dev/developers/general/github-actions/.
     bugs, and other work tracking
 *   [IREE Discord server](https://discord.gg/wEWh6Z9nMU): Daily development
     discussions with the core team and collaborators
-*   [iree-discuss email list](https://groups.google.com/forum/#!forum/iree-discuss):
-    Announcements, general and low-priority discussion
+*   [iree-announce email list](https://lists.lfaidata.foundation/g/iree-announce):
+    Announcements
+*   [iree-technical-discussion email list](https://lists.lfaidata.foundation/g/iree-technical-discussion):
+    General and low-priority discussion
```

Comment:
We could keep links to the old list for some time while conversations move over. Another idea is to send a message to the new list linking to the old list.

Reply:
I was just about to re-add the link to the old mailing list to the README but probably sending an email to the new list (or rather to the new lists) which is then available persistently in the archive might even be more useful. We could also do both. WDYT?

Reply:
How about we keep both links for ~1 month, but label which is new and old?

Something like this?

```markdown
*   (New) [iree-announce email list](https://lists.lfaidata.foundation/g/iree-announce):
    Announcements
*   (New) [iree-technical-discussion email list](https://lists.lfaidata.foundation/g/iree-technical-discussion):
    General and low-priority discussion
*   (Old) [iree-discuss email list](https://groups.google.com/forum/#!forum/iree-discuss):
    Announcements, general and low-priority discussion
```

"Old" could also be "archive", "deprecated", "legacy", or some other word

Reply:
Thanks for that suggestion. I add this with the prefix "legacy" to the readme, as well to the `index.md` of the website.